File: 7-kyu/remove-consecutive-duplicate-words/python/solution.py
def remove_consecutive_duplicates(s):
    if not s:
        return ''
    s = s.split()
    ret = [s[0]]
    while s:
        if s[0] != ret[-1]:
            ret.append(s[0])
        s.pop(0)
    return ' '.join(ret)
        

File: 7-kyu/remove-consecutive-duplicate-words/javascript/solution.js
const removeConsecutiveDuplicates = (s) => {
  if (!s) {
    return "";
  }
  s = s.split(" ");
  let ret = [s[0]];
  while (typeof (_ = s.shift()) !== 'undefined') {
    if (s[0] !== ret[ret.length - 1]) {
      ret.push(s[0]);
    }
  }
  return ret.slice(0, -1).join(" ");
};
